### PR TITLE
added HTML Assignment

### PR DESCRIPTION
**Pull request by shivani-pawbake**

### CODE REVIEW

File: Shivani/HTML/firstassignment.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
-<head>
+ <head>
```
Comment (akash19972306) [inline Shivani/HTML/firstassignment.html:3]: you are using double <head > tag

File: Shivani/HTML/firstassignment2.html [hunk 2]
```diff
@@ -5,8 +5,6 @@
 <h1> HTML</h1>
 <h2>Introduction</h2>
 <p> HTML is a standard markup language for Web Pages.</p>
-
-
 <h2>HTML Editor</h2>
 <h3>Notepad or TextEdit.</h3>
 <p> Web pages can be created and modified by using professional HTML editors.<br>However, for learning HTML we recommend a simple text editor like, Notepad(PC) or TextEditor(Mac).</p>
```
Comment (akash19972306) [inline Shivani/HTML/firstassignment2.html:10]: use <br>tag to break the line

**Review by akash19972306 — CHANGES_REQUESTED** [review]

File: Shivani/HTML/firstassignment4.html [hunk 3]
```diff
@@ -1,9 +1,7 @@
 <!DOCTYPE html>
-
 <html>
 <head>
 <title>assign4</title>
-
 </head>
 <body bgcolor="lightblue">>
 <h1 style="color:tomato;"> Heading1 </h1>
```
Comment (vinod2406) [inline Shivani/HTML/firstassignment4.html]: in assignment 4 you have tell about yourself not just tags adding from internet ok

**Review by vinod2406 — CHANGES_REQUESTED** [review]

File: Shivani/HTML/firstassignment2.html (hunk 2 above)
Comment (pooja-nagare) [reply Shivani/HTML/firstassignment2.html:10]: <br> tag is used already

**Comment by vinod2406** [comment]
bt sir tell na
that you have to add about yourself that is like self introduction but
there is nothing in you file..just tags

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**